fission-core/library/Fission/Web/Auth/Token/UCAN/Potency/Types.hs

```diff
@@ -12,30 +16,30 @@ append-only rights, so
 
 -- | How much power allowed in an authorization
 data Potency
-  = AuthNOnly   -- ^ Read signature only -- just a proof. Cannot delegate further.
```

Comment:
I removed the `AuthNOnly` constructor here, in favor of using `Maybe Potency` instead of `Potency` everywhere.
The reason is that in `hs-ucan` we don't want to depend on `Potency` (because it's also fission-specific), but we *need* to decode to *some* potency in case the `ptc` key is missing completely.

Reply:
Yeah, for this version of UCAN totally.

Thinking out loud: as we bring things to up date, I imagine that we'll have something like...

```haskell
data Capabilities custom
  = None
  |  UserDefined custom
  | AllByParenthood -- e..g Linking
```

...but obviously we need to actually solidify that in the v0.8 spec before we do that.